Drivers/CacheControlDriver.cs

```diff
@@ -87,7 +87,7 @@ private ICacheControl GetCacheControl(CacheControlPart cacheControlPart)
 
             if (cacheControlPart.UseDefault && defaultSettings != null)
             {
-                cacheControl = defaultSettings;
+                cacheControl = defaultSettings.Result;

```

Comment:
Instead of using `.Result`, you should add an `await` to the call on line 86, and then change this method to be asynchronous.